docker/ansible-in-docker/build.bash

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -vx
+
+function buildAnsible {
+    dockerTag=oso-ansible:${ansibleVersion}
+
+    bashScript=./oso-ansible-playbook-${ansibleVersion}
+
+    docker build --tag ${dockerTag} - << EndOfDockerfile
+FROM fedora:25
```

Comment:
This needs to support building both rhel7 and centos7 containers.

See our other containers for the patter.